### DISCUSSION TITLE
[categorie_corrigee] Cas 6 & 11: Type de vélo VAE

### DISCUSSION THREAD

**Discussion by JMPivette** [JMPivette/evavelo#11] [Q&A] [answered]
Dans la méthodologie, pour le cas 6 on parle d'un type de vélo "VAE" comme critère pour affecter la catégorie corrigée.

Pour l'instant j'ai considéré que cela voulait dire que tous les vélos du groupe étaient des VAE (nb_vae = nb_total_vélo).

Est-ce correct? 
Ou faut-il juste un VAE dans le groupe? Voire une majorité de VAE?

**Comment by SteffMangin** [comment, marked as answer]
On va partir du principe que c'est tout le groupe
